diff --git a/BUILD.bazel b/BUILD.bazel
index 750ac5bba53..b87c70a0275 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -1,10 +1,7 @@
-load("@aspect_rules_js//js:defs.bzl", "js_library")
-load("@aspect_rules_js//npm:defs.bzl", "npm_link_package")
 load("@bazel_skylib//lib:selects.bzl", "selects")
 load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
 load("@capnp-cpp//src/capnp:cc_capnp_library.bzl", "cc_capnp_library")
 load("@npm//:capnp-es/package_json.bzl", capnp_es_binary = "bin")
-load("@npm//:defs.bzl", "npm_link_all_packages")
 
 cc_capnp_library(
     name = "icudata-embed",

